File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--complex-data-binding-in-a-document-level-project_3.cs
        private void button1_Click(object sender, EventArgs e)
        {
            try
            {
                this.employeesTableAdapter.Update(this.northwindDataSet.Employees);
            }
            catch (System.Data.DataException ex)
            {
                MessageBox.Show(ex.Message);
            }
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--complex-data-binding-in-a-document-level-project_3.vb
    Private Sub Button1_Click(ByVal sender As Object, ByVal e As System.EventArgs) _
        Handles Button1.Click

        Try
            Me.EmployeesTableAdapter.Update(Me.NorthwindDataSet.Employees)

        Catch ex As System.Data.DataException
            MessageBox.Show(ex.Message)
        End Try
    End Sub